db/migrations/20230830144100_improve_blocks_bls_change_pubkey_text_idx.sql

```diff
@@ -0,0 +1,24 @@
+-- +goose Up
```

Comment:
Missing CONCURRENTLY for dropping & creating an index.

Reply:
done